specs.json

```diff
@@ -87,6 +87,7 @@
       "passDtmf": "boolean",
       "playBeep": "boolean",
       "disableBidirectionalAudio": "boolean",
+      "bidirectionalAudioSampleRate": "number",
```

Comment:
till now, bidirectional audio has meant they send us base64 encoded audio, we write it to a file and play that file.  That feature needs to stay in place.  So we need twop new properties: a boolean to indicate streaming of the audio is desired, and the sample rate.  I think it might be best to have 
```
"listenOptions" : {
    ...
    "disableBidrectionalAudio": false,
     "bidirectionalAudio: {
           "streaming": true,
           "sampleRate: 8000
     }
```
so "bdirectionalAudio" is an optional property, if not provided or if streaming = false you get the legacy behavior

Reply:
Hi @davehorton 

when streaming == true,

If the user still send playAudio command with audio base64 text data to jambonz, do we still play that audio parallel with streaming data?

Reply:
no if streaming === true then discard base64 audio